playwright/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "compilerOptions": {
-        "target": "es2016",
+        "target": "es2018",
```

Comment:
for named capture groups in regex, should be okay ... I think?

Reply:
Yeah its fine